pretyping/glob_ops.mli

```diff
@@ -111,4 +111,12 @@ val glob_constr_of_cases_pattern : Environ.env -> 'a cases_pattern_g -> 'a glob_
 val add_patterns_for_params_remove_local_defs : Environ.env -> constructor ->
   'a cases_pattern_g list -> 'a cases_pattern_g list
 
+(** Pretty-printer hook: [print_glob_constr_env env c] will pretty
+   print c if the pretty printing layer has been linked into the Coq
+   binary. *)
+val print_glob_constr_env : Environ.env -> glob_constr -> Pp.t
+
+(** [set_print_glob_constr f] sets f to be the pretty printer *)
+val set_print_glob_constr : (Environ.env -> glob_constr -> Pp.t) -> unit
```

Comment:
Shouldn't this be in an Internal module like the termops printers?
Also why use ref instead of Hook?